content/zh-cn/_index.md

```diff
@@ -5,15 +5,15 @@ title: "云原生词汇表"
 
 # 云原生词汇表
 
-云原生词汇表是由 CNCF 商业价值小组委员会 (BVS, Business Value Subcommittee) 领导的一个项目。该项目目的是在不需要先验技术知识的情况下，以清晰通俗的语言解释云原生概念。
+云原生词汇表是由 CNCF 商业价值小组委员会 (BVS, Business Value Subcommittee) 领导的一个项目。该项目的目的是在不需要任何先决技术知识的情况下，以通俗易懂的语言阐释云原生概念。
 
 <p><img class="mt-5" src="/images/homepage/stage.jpg" alt="A woman and two men presenting technical info on a stage"></p>
 
 ## 社区贡献
 
-对本机词汇表的更改、添加和改进对所有人开放。我们使用由 CNCF 管理的社区驱动流程来开发和完善这个共享词典。该词汇表提供了一个厂商中立的平台，用于管理云原生技术的共享词汇表。欢迎所有参与者根据本项目的目的和章程作出贡献。
+我们欢迎所有人对本词汇表提出修改、添加和改进建议。我们采用了 CNCF 所管理的社区驱动流程来开发和改进这个共享词典。该词汇表提供了一个厂商中立的平台，用于管理云原生技术的共享词汇表。欢迎所有参与者在遵守本项目的目的和章程的前提下作出贡献。
 
-任何想要贡献的人都可以提交 GitHub issue 或 创建 PR。欲了解更多信息，请查看 [如何贡献](/zh-cn/contribute/) 和遵循 [样式指南](/zh-cn/style-guide/)，并加入 CNCF Slack 中的 #glossary 频道。这里有 #glossary-localizations 频道，供想要将 glossary 翻译成母语的人使用。
+任何想要贡献的人都可以提交 GitHub Issue 或 创建 PR。欲了解更多信息，请查看 [如何贡献](/zh-cn/contribute/) 和遵循 [样式指南](/zh-cn/style-guide/)，并加入 CNCF Slack 中的 #glossary 频道。这里还有 #glossary-localizations 频道，供想要将 glossary 翻译成母语的人使用。
```

Comment:
```suggestion
任何想要贡献的人都可以提交 GitHub issue 或 创建 PR。欲了解更多信息，请查看 [如何贡献](/zh-cn/contribute/) 和遵循 [样式指南](/zh-cn/style-guide/)，并加入 CNCF Slack 中的 #glossary 频道。这里还有 #glossary-localizations 频道，供想要将 glossary 翻译成母语的人使用。
```
I think this expression should be consistent with the English version. Need other approvers to review together